brayns/tasks/UploadBinaryTask.cpp

```diff
@@ -121,7 +122,8 @@ void UploadBinaryTask::_checkValidity(EnginePtr engine)
         if (param.type.empty() || param.size == 0)
             throw MISSING_PARAMS;
 
-        if (!registry.isSupported(param.type))
+        if (!registry.isSupported(param.type) &&
+            !isSupportedArchiveType(param.type))
```

Comment:
Hmm instead of special handling here (which applies to uploads only?) couldn't this be handled by the registry? What about opening zip files from a filesystem path (not uploaded)?

Reply:
I wanted do that initially, but discarded it at some point. For the remote location, the user can unpack to a folder himself :)
But I'll look at it again, it should be possible, unless I really find the blocker why I didn't do it in the end.